ci/deploy.sh

```diff
@@ -56,7 +56,9 @@ function cargo_publish {
 	# NOTE(nlordell): For some reason, the next publish fails on not being able
 	#   to find the new version; maybe it takes a second for crates.io to update
 	#   its index
-	sleep 10
+	if [[ "$1" != "." ]]; then
```

Comment:
If we wanted to be more sophisticated, maybe we could do a request to crates.io to see if the package is there?

Reply:
That's the next step if this continues to fail.